lib/task/pdf/ead-pdf-inventory-summary.xsl

```diff
@@ -1128,8 +1064,8 @@
     <xsl:template match="ead:note">
         <xsl:choose>
```

Comment:
Could the <xsl:apply-templates> happen after the end of the choose statement? It looks like it always gets fired, no matter what condition within the choose structure, but I am not versed in this stuff so I'm probably wrong. ;)

Reply:
@mcantelon I think you're right that <xsl:apply-templates/> should come after the <choose> to avoid repetition. :)